src/main/js/wiki/components/PageHeader.tsx

```diff
@@ -178,17 +183,29 @@ class PageHeader extends React.Component<Props, State> {
       />
     );
 
+    const branch_entries = [];
+    for (const branch of this.props.branches) {
+      branch_entries.push({ name: branch.name, link: branch._links.self.href });
```

Comment:
The `link` field seems unused.

Reply:
removed the usage of link as it is not used anywhere